backend/apis/parse-docket-pdf.api.js

```diff
@@ -33,8 +33,8 @@ app.post("/api/docket-pdfs", (req, res) => {
             const processJson = req.query.processJson === "true" ? true : false;
             if (processJson) {
               //comment this back in if you want to test flag json
-              //res.send(generateFlagJson(parsePdfText(thePdf.text)));
-              res.send(parsePdfText(thePdf.text));
+              res.send(generateFlagJson(parsePdfText(thePdf.text)));
+              // res.send(parsePdfText(thePdf.text));
```

Comment:
for now; let's leave this line commented out in master.

Reply:
I have changed this in the new commit.